.travis.yml

```diff
@@ -1,19 +1,24 @@
 language: python
 python:
-- '3.5'
 - '3.6'
 jdk:
   - openjdk8 # recommended jdk for solr < 7
 env:
   global:
-    - SOLR_VERSION=6.6.5
     - JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk-amd64  # otherwise jdk11 will be used
   jobs:
-    - DJANGO=
-    - DJANGO=1.11
-    - DJANGO=2.0
-    - DJANGO=2.1
-    - DJANGO=2.2
+    # solr 8
+    - SOLR_VERSION=8.6.2 DJANGO=
+    - SOLR_VERSION=8.6.2 DJANGO=1.11
+    - SOLR_VERSION=8.6.2 DJANGO=2.0
+    - SOLR_VERSION=8.6.2 DJANGO=2.1
+    - SOLR_VERSION=8.6.2 DJANGO=2.2
+    # solr 6
+    - SOLR_VERSION=6.6.5 DJANGO=
+    - SOLR_VERSION=6.6.5 DJANGO=1.11
+    - SOLR_VERSION=6.6.5 DJANGO=2.0
+    - SOLR_VERSION=6.6.5 DJANGO=2.1
+    - SOLR_VERSION=6.6.5 DJANGO=2.2
```

Comment:
is there a reason we don't test against 7?

Reply:
We're not planning to use it ourselves, so why expand the matrix and slow the tests down? We're not aware of anyone using it, and hopefully someone new to parasolr would start with solr 8. We could always add it to our testing matrix if someone requests (and makes the case).

Reply:
wouldn't it add close to zero time if the matrix is running in parallel? seems odd to say we support it if we don't test for it. I guess we could say "supports solr>6; tested on solr 6 and 8"

Reply:
i guess we don't actually say "supports" in the README at all, which is smart. so maybe this isn't an issue :)